pydocx/parsers/Docx2Html.py

```diff
@@ -32,6 +32,14 @@ def parsed(self):
         )
         return content
 
+    @property
+    def parsed_without_head(self):
+        content = super(Docx2Html, self).parsed
+        content = "<html><body>{body}</body></html>".format(
+            body=content,
+        )
+        return content
```

Comment:
The preferred pattern for parser customization is to extend the class in your own code, and override/replace methods as desired. For example, you could do the following (we do something similar):

```python
class OSFDocx2HtmlParser(Docx2Html):
    def head(self):
        return ''
```

Then you would only need to do create an instance of `OSFDocx2HtmlParser` and get the `parsed` value.

How does this approach sound?